### PR TITLE
Update SVS README.md

### PR DESCRIPTION
**Pull request by South-Twilight**
This PR updates README.md in SVS task.

### CODE REVIEW

File: egs2/TEMPLATE/svs1/README.md [hunk 1]
```diff
@@ -0,0 +1,540 @@
...
+    ```
+    
+- `label` format
+    ```
+    uttidA (startA1, endA1, phA1) (startA2, endA2, phA1) ...
+    uttidB (startB1, endB1, phB1) (startB2, endB2, phB2) ...
+    ...
+    ```
+    
+- `midi.scp` format
```
Comment (ftshijt) [inline egs2/TEMPLATE/svs1/README.md:418]: Can you double-check with @A-Quarter-Mile whether this is still hold?

**Review by ftshijt — COMMENTED** [review]
Thanks! @A-Quarter-Mile , could you also check the readme to see if there is any issue with the description (especially for the data preparation and some preprocessing techniques)

@South-Twilight Could you also add a link in the espnet/README.md for this detailed doc?

File: egs2/TEMPLATE/svs1/README.md [hunk 2]
```diff
@@ -0,0 +1,540 @@
...
+```sh
+$ ./run.sh \
+    --stage 1 \
+    --stop_stage 4 \
+```
+Second, check "train_config" (default `conf/train.yaml`), "score_feats_extract" (*syllabel level* in XiaoiceSing) and modify "vocoder_file" with your own vocoder path.
+```sh
+$ ./run.sh --stage 5 \
+    --train_config conf/tuning/train_naive_rnn.yaml \
+    --score_feats_extract frame_score_feats \
```
Comment (A-Quarter-Mile) [inline egs2/TEMPLATE/svs1/README.md:225]: --score_feats_extract syllable_score_feats \

File: egs2/TEMPLATE/svs1/README.md [hunk 3]
```diff
@@ -0,0 +1,540 @@
...
+    dump/raw/eval/gt_wavdir_or_wavscp.scp
+```
+```sh
+cd egs/<recipe_name>/svs1
+. ./path.sh
+# Evaluate log-F0 RMSE & Semitone ACC & VUV Error Rate
+./pyscripts/utils/evaluate_f0.py \
+    exp/<model_dir_name>/<decode_dir_name>/eval/wav/gen_wavdir_or_wavscp.scp \
+    dump/raw/eval/gen_wavdir_or_wavscp.scp
+```
```
Comment (A-Quarter-Mile) [inline egs2/TEMPLATE/svs1/README.md:368]: We've changed it into three separate files for log-F0 RMSE & Semitone ACC & VUV Error Rate.

File: egs2/TEMPLATE/svs1/README.md [hunk 4]
```diff
@@ -0,0 +1,540 @@
...
+./run.sh
+```
+
+- Directory structure
+    ```
+    data/
+    ├── tr_no_dev/     # Training set directory
+    │   ├── text       # The transcription
+    │   ├── label      # Specifying start and end time of the transcription 
+    │   ├── midi.scp   # MIDI file path
```
Comment (A-Quarter-Mile) [inline egs2/TEMPLATE/svs1/README.md:390]: (midi.scp)   # MIDI file path

File: egs2/TEMPLATE/svs1/README.md (hunk 1 above)
Comment (A-Quarter-Mile) [reply egs2/TEMPLATE/svs1/README.md:418]: We can add a note here. It's prepared for recipes with only MIDI for pitch information.

File: egs2/TEMPLATE/svs1/README.md [hunk 5]
```diff
@@ -0,0 +1,540 @@
...
+- [Supported text cleaner](#supported-text-cleaner).
+- [Supported text frontend](#supported-text-frontend).
+
+Data preparation will end in stage 4. You can skip data preparation (stage 1 ~ stage 4) via `--skip_data_prep` option.
+
+### 5. SVS Statistics collection
+
+Statistics calculation stage.
+It collects the shape information of the input and output and calculates statistics for feature normalization (mean and variance over training and validation sets).
+
```
Comment (A-Quarter-Mile) [inline egs2/TEMPLATE/svs1/README.md:84]: store pitch (and feats? to be discussed) in stage 5 via `--write_collected_feats true`

File: egs2/TEMPLATE/svs1/README.md [hunk 6]
```diff
@@ -0,0 +1,540 @@
...
+## Recipe flow
+
+SVS recipe consists of 8 stages.
+
+### 1. Database-dependent data preparation
+
+Data preparation stage.
+
+It calls `local/data.sh` to creates Kaldi-style data directories but with additional `midi.scp` and `label` in `data/` for training, validation, and evaluation sets.
```
Comment (A-Quarter-Mile) [inline egs2/TEMPLATE/svs1/README.md:41]: replace 'midi.scp' with 'score.scp' here.

File: egs2/TEMPLATE/svs1/README.md (hunk 1 above)
Comment (South-Twilight) [reply egs2/TEMPLATE/svs1/README.md]: I can't find where to add a link in espnet/README.md.

Comment (ftshijt) [reply egs2/TEMPLATE/svs1/README.md]: That's no problem. I can add that later

File: egs2/TEMPLATE/svs1/README.md [hunk 7]
```diff
@@ -0,0 +1,550 @@
...
+
+This is a template of SVS recipe for ESPnet2.
+
+## Table of Contents
+
+* [ESPnet2 SVS Recipe TEMPLATE](#espnet2-svs-recipe-template)
+  * [Table of Contents](#table-of-contents)
+  * [Recipe flow](#recipe-flow)
+    * [1\. Database-dependent data preparation](#1-database\-dependent-data-preparation)
+    * [2\. Standard audio and midi formatting](#2-standard-audio-and-midi-formatting)
```
Comment (A-Quarter-Mile) [inline egs2/TEMPLATE/svs1/README.md:11]: The subtitles in this part should be consistent with the following content so as to build the links.

File: egs2/TEMPLATE/svs1/README.md [hunk 8]
```diff
@@ -0,0 +1,550 @@
...
+See also:
+- [About data directory](#about-data-directory)
+
+### 2. Wav dump / Embedding preparation
+
+If you specify `--feats_type raw` option, this is a wav dumping stage which reformats `wav.scp` in data directories.
+
+Else, if you specify `--feats_type fbank` option or `--feats_type stft` option, this is a feature extracting stage (to be updated).
+
+MIDI is also normalized and segmented at this stage.
```
Comment (A-Quarter-Mile) [inline egs2/TEMPLATE/svs1/README.md:52]: We can remove MIDI here.

File: egs2/TEMPLATE/svs1/README.md [hunk 9]
```diff
@@ -0,0 +1,550 @@
...
+- [Supported text frontend](#supported-text-frontend).
+
+Data preparation will end in stage 4. You can skip data preparation (stage 1 ~ stage 4) via `--skip_data_prep` option.
+
+### 5. SVS Statistics collection
+
+Statistics calculation stage.
+It collects the shape information of the input and output and calculates statistics for feature normalization (mean and variance over training and validation sets).
+
+In this stage, you can set `--write_collected_feats true` to store statistics of pitch.
```
Comment (A-Quarter-Mile) [inline egs2/TEMPLATE/svs1/README.md:85]: and feats?

File: egs2/TEMPLATE/svs1/README.md [hunk 10]
```diff
@@ -0,0 +1,550 @@
...
+We provide four objective evaluation metrics:
+
+- Mel-cepstral distortion (MCD)
+- Logarithmic rooted mean square error of the fundamental frequency (log-F0 RMSE)
+- Semitone accuracy (Semitone ACC)
+- Voiced / unvoiced error rate (VUV_E)
+
+For MCD, we apply dynamic time-warping (DTW) to match the length difference between ground-truth singing and generated singing.
+
+Here we show the example command to calculate objective metrics:
```
Comment (A-Quarter-Mile) [inline egs2/TEMPLATE/svs1/README.md:360]: If we don't list four metrics separately, maybe we can summarize them into evaluate_*.py.

**Comment by A-Quarter-Mile** [comment]
@jerryuhoo Do you need any extra explanation for the new vocoders in README this time?

**Comment by ftshijt** [comment]
Could you please not  mix the PR with other features? For this PR, let's only focus on the readme section
